pkg/testgridanalysis/testgridconversion/to_raw_data.go

```diff
@@ -116,6 +116,54 @@ func isOverallTest(testName string) bool {
 	return testName == overall || strings.HasSuffix(testName, ".Overall")
 }
 
+//Specific set of test names that include random characters in them
+//if they match the known start then look to replace the random chars with 'random'
+var matchRandomStart = regexp.MustCompile("^\"Installing \"Red Hat Integration")
+var matchRandom = regexp.MustCompile("operator in test-[a-z]+")
+var matchRandomReplace = "operator in test-random"
+
+func removeRandomOperatorTestNames(testName string) string {
+	//not necessary but narrowing the scope we apply the replace all to
+	//by verifyting the name starts with our known case
+	match := matchRandomStart.MatchString(testName)
+	if match {
+		return matchRandom.ReplaceAllString(testName, matchRandomReplace)
+	}
+	//essentially an else
+	return testName
+}
+
+func fixOldStyleTestNames(testName string) string {
+	// we override some test names based on their type.  Historically we misnamed the install and upgrade tests
+	// what we really want is to call these the final state
+	// This prevents any real results with this junit from counting.  This should only be needed during our transition  and
+	// we have to keep it to interpret historical results from 4.6.
+	if testidentification.IsOldInstallOperatorTest(testName) || testidentification.IsOldUpgradeOperatorTest(testName) {
+		operatorName := testidentification.GetOperatorNameFromTest(testName)
+		testName = testgridanalysisapi.OperatorFinalHealthPrefix + " " + operatorName
+	}
+
+	return testName
+}
+
+//function to group together calls for cleansing the test names as needed
```

Comment:
The convention is that the comment for a function begins with the name of the function.  So you can phrase it like: `// cleanTestName groups together calls for ...`